package.json

```diff
@@ -19,7 +19,8 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.12.5"
+    "mongoose": "^5.12.5",
+    "react-icons": "^4.2.0"
```

Comment:
Neither of these packages should be added at this point. `mongoose` is for the backend, which you should develop at a later point, `react-icons` are for the frontend, so I's expect them in `/client/package.json` and not here